app/features/feedback/shared/strategies/drawing/pointInEllipse/reducer.js

```diff
@@ -0,0 +1,34 @@
+function isAnnotationWithinEllipse(rule, annotation) {
+  const annotationX = annotation.x;
+  const annotationY = annotation.y;
+  const feedbackX = rule.x;
+  const feedbackY = rule.y;
+  const feedbackA = rule.tolerance_a;
+  const feedbackB = rule.tolerance_b;
+  const feedbackTheta = rule.theta;
+
+  const feedbackThetaRad = Math.PI * (rule.theta / 180.0)
+  const projectedX = (annotationX - feedbackX) * Math.cos(feedbackThetaRad) + (annotationY - feedbackY) * Math.sin(feedbackThetaRad)
+  const projectedY = (annotationY - feedbackY) * Math.cos(feedbackThetaRad) - (annotationX - feedbackX) * Math.sin(feedbackThetaRad)
+
+  // Math.pow is a restricted property, but using the exponential operator (**)
+  // breaks the build :(
```

Comment:
Not a blocker, but I'm curious about why using ** was crashing the app?

Reply:
Not sure. I think either Mark or Roger wrote the original code and left that comment.